packages/flutter/lib/src/material/circle_avatar.dart

```diff
@@ -67,13 +69,19 @@ class CircleAvatar extends StatelessWidget {
   /// The color with which to fill the circle. Changing the background
   /// color will cause the avatar to animate to the new color.
   ///
-  /// If a background color is not specified, the theme's primary color is used.
+  /// If a background color is not specified, the theme's light primary color
+  /// is used with dark foreground colors, and the dark primary color with light
```

Comment:
Although it's a bit clunky to write, it would help to write [ThemeData.primaryColorDark] instead of "dark primary color" etc.

Reply:
OK.